src/quacc/types.py

```diff
@@ -759,6 +760,24 @@ class VaspASEOptSchema(VaspSchema, OptSchema):
         """Type hint associated with VASP relaxations run via ASE"""
 
     # ----------- Recipe (Espresso) type hints -----------
+    class SystemData(TypedDict):
+        occupations: str
+        smearing: str
+        degauss: float
+
+    class ElectronsData(TypedDict):
+        conv_thr: float
+        mixing_mode: str
+        mixing_beta: float
+
+    class InputData(TypedDict):
+        system: SystemData
+        electrons: ElectronsData
+        control: NotRequired[dict[str, Any]]
+
+    class BaseSet(TypedDict):
```

Comment:
Can we rename this `EspressoBaseSet` since I imagine there will be other similar dictionaries for other calculators.

Reply:
Done !

Reply:
Thanks!